app/Console/Commands/RosterUpdate.php

```diff
@@ -39,7 +39,7 @@ public function __construct() {
      */
     public function handle() {
         $client = new Client();
-        $res = $client->get('https://api.vatusa.net/v2/facility/'.Config::get('vatusa.facility').'/roster?apikey='.Config::get('vatusa.api_key'));
+        $res = $client->get(Config::get('vatusa.base').'/v2/facility/'.Config::get('vatusa.facility').'/roster?apikey='.Config::get('vatusa.api_key'));
```

Comment:
See comment above - move to separate issue.

Reply:
Removed